developer-docs/README.md

```diff
@@ -1,48 +1,55 @@
+  * Managerial
+    * To be used ideally for project managers and team leaders, the role hides most technicalities  within projects and other capabilities provided which have of no use for project management.
+  * Technical
+    * Provides more specific actions on the functioning of projects and its assets, while hiding project management services.
 
-We recommend you have one per environment, with each namespace prefixed with the project name.
-For example if the project name were *boomba* the namespaces might be *boomba-dev*, *boomba-uat*, *boomba-preprod*.
-
-You will also need to ask for your team to be granted access to the new namespaces, which you can do by making a request here:  
-https://github.com/UKHomeOffice/application-container-platform-bau/projects/1
-
-## Project getting to production guide
-Content to follow soon! In the meantime please ask for help in the Devops channel on the [HOD DSP Slack](https://hod-dsp.slack.com)
+Updates are also on its way to include more self-service tools, along with documentation, FAQs and live status updates.
+> Access to the Platform Hub requires the `ACP Platform` VPN profile. Please follow the [Developer getting started guide](#Developer-getting-started-guide) for instructions on connecting to VPNs.
 
 ## Developer getting started guide
-[This guide](dev_setup.md) shows you how to:
+Some prerequisites are required before developing on ACP. [This guide](dev_setup.md) will show you how to:
 
-* Get access to the VPN that allows you to connect to the platform
+* Get access to the VPNs that allows you to connect to the platform
 * Get setup with the kubectl client that lets you deploy applications to kubernetes
 * Get access to Quay and Artifactory
 
-## Developer application deployment guide
+## Developer Platform introduction guide
 [This guide](platform_introduction.md) takes you through deploying a demo application to the platform, and explains some basic steps you can take to debug applications on the platform
 
+## Project getting started guide
+Kubernetes resources (including your applications!) are always deployed into a particular namespace.
+These namespaces provide separation between the different projects hosted on the platform.
+We also recommend that each project has one namespace per environment. For example if the project name were *foo* the namespaces might be *foo-dev*, *foo-uat*, *foo-preprod*.
+
+For new projects you will need to create new namespaces to house your applications. Instructions on doing so can be found in the [How to docs](../how-to-docs/README.md)
+
+You will also need to ask for your team to be granted access to the new namespaces, which you can do by submitting a BAU ticket here:
+https://github.com/UKHomeOffice/application-container-platform-bau/issues <br/>
```

Comment:
1. We should not be pointing directly to the BAU board from now on – let's use specific support request form links in the hub.
1. I reckon you can combine this paragraph with the one above, since you already point to the "how to" docs (where the kube token how to doc resides).